src/index.js

```diff
@@ -49,4 +30,62 @@ const listFooter = `</ul>
   <button type="button">Clear all completed</button>
 </div>`;
 
-document.querySelector('.content-1-1').innerHTML = listHead + data.map((item) => listItem(item)).join('') + listFooter;
+const display = () => {
+  document.querySelector('.content-1-1').innerHTML = listHead + data.map((item) => listItem(item)).join('') + listFooter;
+
+  document.querySelectorAll('.value').forEach((input, i) => {
+    input.onclick = (e) => {
+      const activeElement = document.querySelector('.active');
+
+      if (activeElement) {
+        const icons = [...activeElement.childNodes].filter((icon) => icon.innerHTML === 'more_vert' || icon.innerHTML === 'delete');
+
+        icons[0].classList.remove('hide');
+        icons[1].classList.add('hide');
+
+        activeElement.classList.remove('active');
+      }
+      e.target.parentElement.parentElement.classList.add('active');
+      const icons = [...e.target.parentElement.parentElement.childNodes].filter((icon) => icon.innerHTML === 'more_vert' || icon.innerHTML === 'delete');
+      icons[0].classList.add('hide');
+      icons[1].classList.remove('hide');
+
+      icons[1].onclick = () => {
+        if (!input.value) {
+          data = removeTask(i, data);
+
+          display();
+        }
+      };
```

Comment:
- [ ] The remove functionality is not working as expected when the delete icon is clicked. Kindly work on it so a todo is removed when the delete icon is clicked 👍🏽

Reply:
From the demo, this is how the remove button works. You must first delete the text, then click on the delete button. I reproduced exactly what the demo and the site do.
https://www.youtube.com/watch?v=AcUd-_Yjjqg&ab_channel=EmiliaAndrzejewska
https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/